app/serializers/api/v1/offers_package_serializer.rb

```diff
@@ -0,0 +1,23 @@
+module Api::V1
+  class OffersPackageSerializer < ApplicationSerializer
+    embed :ids, include: true
+    attributes :id, :offer_id, :package_id
+    has_one :offer, serializer: OfferShallowSummarySerializer
+
+    def offer_id
+      object.offer_id
+    end
+
+    def offer_id__sql
+      "offer_id"
```

Comment:
@abulsayyad123 Can you please explain why we need to explicitly specify `offer_id__sql` and `package_id__sql`? as those are the columns of  table 

https://github.com/crossroads/api.goodcity/blob/62ae2179c4f52957581fcbf94fb8319ba1bf8e25/db/schema.rb#L422-L425

Reply:
Yes.. we can remove this. At first I was renaming `package` to `item` that is why I had added this `__sql` methods.